tools/docker/builder/openwrt/toolchains/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ubuntu:16.04
+
+RUN dpkg --add-architecture i386
+
+RUN apt-get -y update && apt-get install --no-install-recommends -y sudo git-core \
+gawk wget unzip make uml-utilities bridge-utils debootstrap gcc-multilib \
+perl automake autoconf fakeroot build-essential crash makedumpfile \
+kernel-wedge git-core libncurses5 libncurses5-dev libelf-dev asciidoc binutils-dev \
+bison imagemagick openssl cmake git-core expect xutils-dev genext2fs m4 qemu \
+libqt4-dev qt4-dev-tools python-setuptools \
```

Comment:
I don't see why we need QT - care to explain?

Reply:
The most usable commands for package or kernel options selection are `make defconfig`, `make menuconfig` and so on which can work in console, but maybe some developers use [xconfig](https://en.wikipedia.org/wiki/Xconfig) for the same purposes. That tool requires Qt

Reply:
IMO the whole interactive option should be removed from the `entrypoint.sh` - developers should prepare everything in the build server natively, and just use a docker container to run `make` - in which case we should limit the tools we install only to what is needed for building prplWrt.
See `tools/docker/builder/rdkb/toolchain/bitbake.sh` for how we do this for building RDKB - I don't think that prplWrt should be any different.

Actually, the only reason we did that for RDKB since it requires Ubuntu 16.04 for some reason and our build server is Ubuntu18.04 LTS which is too new. There is no such requirement for prplWrt for all I know - which raises the question, why do we need this PR in the first place?

Provided we do, I would expect this Dockerfile to be part of prplWrt and not prplMesh, but since for convenience we have this for RDKB then OK, I guess.

If we do need this, then I expect that the following to be done **outside** the container:
```
git clone https://git.prpl.dev/prplmesh/prplwrt.git
cd prplwrt
./scripts/feeds setup "src-git,packages,https://git.openwrt.org/feed/packages.git;openwrt-19.07" "src-git,intel_feed,https://git.prpl.dev/prplmesh/feed-intel.git;intel-19.07" "src-git,iwlwav,https://git.prpl.dev/prplmesh/iwlwav.git;iwlwav-19.07" "src-git,routing,https://github.com/openwrt-routing/packages.git"
./scripts/feeds update 
./scripts/feeds install -a -p intel_feed
./scripts/feeds install -a -p iwlwav
./scripts/feeds install intel_mips
./scripts/feeds install zmq
./scripts/gen_config.sh rax40
```
Not sure about the `gen_config.sh` part though - need to check if it depends on host tools.
Then, like `bitbake.sh`, add a build `tools/docker/builder/prplwrt/make.sh` which will run make from within a container while mapping the prplWrt clone folder, and the home folder if necessary, to the container.